packages/headless/src/features/triggers/insight-trigger-analytics-actions.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  InsightAction,
+  makeInsightAnalyticsActionFactory,
+} from '../analytics/analytics-utils';
+import {SearchPageEvents} from '../analytics/search-action-cause';
+
+export const logNotifyTrigger = (): InsightAction =>
+  makeInsightAnalyticsActionFactory(SearchPageEvents.triggerQuery)(
+    'analytics/trigger/notify',
+    (client, state) => {
+      if (!state.triggers?.notifications.length) {
+        return null;
+      }
+      return client.logTriggerNotify({
```

Comment:
looking at the way you defined this new action in coveo.analytics.js, I see that this action also accepts case metadata, but you are not using them here

Reply:
right good point!